scripts/mage/package.go

```diff
@@ -21,13 +21,24 @@ import (
 	devtools "github.com/elastic/beats/v7/dev-tools/mage"
 )
 
+const opaBundle = "bundle.tar.gz"
+
 // CustomizePackaging modifies the device in the configuration files based on
 // the target OS.
 func CustomizePackaging() {
+	bundleDir := devtools.PackageFile{
```

Comment:
Could you perhaps add the correct opa bundle to our hermit config as well?
https://cashapp.github.io/hermit/usage/management/

Reply:
Weird, they don't have the version we use - **opa-0.42.0**


`hermit install opa-0.42.0`
```
fatal:hermit: opa-0.42.0: https://github.com/cashapp/hermit-packages.git/opa.hcl: no version opa-0.42.0 found in versions (opa-0.30.1, opa-0.38.1, opa-0.43.0, opa-0.44.0) or channels (opa@0, opa@0.30, opa@0.38, opa@0.43, opa@0.44, opa@latest): unknown package
```